connectors/tests/test_byoei.py

```diff
@@ -84,7 +85,17 @@ async def test_prepare_content_index(mock_responses):
     config = {"host": "http://nowhere.com:9200", "user": "tarek", "password": "blah"}
     headers = {"X-Elastic-Product": "Elasticsearch"}
     # prepare-index, with mappings
-    mappings = {"properties": {"name": {"type": "keyword"}}}
+    dynamic_templates = {
+        "data": {
+            "match_mapping_type": "string",
+            "mapping": TEXT_FIELD_MAPPING,
+        }
+    }
+    mappings = {
+        "dynamic": True,
+        "dynamic_templates": dynamic_templates,
+        "properties": {"name": {"type": "keyword"}},
```

Comment:
I see that this was in here before... but I don't think it should be? A `name` property is not in the ruby connectors' mappings, it has `id`.

https://github.com/elastic/connectors-ruby/blob/main/lib/utility/elasticsearch/index/mappings.rb#L92

```suggestion
        "properties": {"id": {"type": "keyword"}},
```

Reply:
It should be fine as this specific function allows arbitrary mappings and this test verifies that these arbitrary mappings are passed correctly.

If we wanna test that our default mappings are passed properly, it should be in a different place, the one that calls `prepare_content_index`